internal/cephfs/controllerserver.go

```diff
@@ -1066,3 +1066,19 @@ func deleteSnapshotAndUndoReservation(
 
 	return nil
 }
+
+// ControllerPublishVolume publish volume on node.
```

Comment:
Why include this in CephFS if it does not need it? Keep it in the `DefaultControllerServer` instead.